src/Xamarin.Android.Build.Tasks/Properties/Resources.resx

```diff
@@ -398,6 +398,10 @@ In this message, the term "binding" means a piece of generated code that makes i
     <value>Using the DX DEX Compiler is deprecated. Please update `$(AndroidDexTool)` to `d8`.</value>
     <comment>The following are literal names and should not be translated: D8, DEX, `$(AndroidDexTool)`, `d8`</comment>
   </data>
+  <data name="XA1023_DOTNET" xml:space="preserve">
+    <value>Using the DX DEX Compiler is not supported in Xamarin.Android projects that target .NET 5 or higher. Please update `$(AndroidDexTool)` to `d8`.</value>
+    <comment>The following are literal names and should not be translated: D8, DEX, Xamarin.Android, .NET 5, `$(AndroidDexTool)`, `d8`</comment>
```

Comment:
It's arguably a little weird to do it since it will temporarily make this message less similar to the existing XA1023 message, but I'll suggest using the "new" wording approach for how to change MSBuild properties:

```
<value>Using the DX DEX Compiler is not supported in Xamarin.Android projects that target .NET 5 or higher. Please set the DEX compiler to 'd8' in the Visual Studio project property pages or edit the project file in a text editor and set the 'AndroidDexTool' MSBuild property to 'd8'.</value>
<comment>The following are literal names and should not be translated: DX, DEX, Xamarin.Android, .NET 5, d8, AndroidDexTool</comment>
```

My thought is that the original XA1023 message can be updated to follow this new wording style too. If it's simplest, that change could be included in this PR, but I can also add it in a separate PR if that'll help keep this PR more on-topic.

Admittedly, I'm not sure if we know yet whether the IDE will display the DEX compiler option in the same way in the project property pages for Android projects in the .NET 5+ project system, but we can update that wording later if needed.

Reply:
👍 I updated both `XA1023` messages.